analysis_service/base/templates/analysis_service/boot-script.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
This file hasn't really changed from the [previous version](https://github.com/mozilla/telemetry-server/blob/master/http/analysis-service/templates/boot-script.sh) so it should be good.